service/manager/manager.go

```diff
@@ -6,9 +6,11 @@ import (
 
 // Manager is responsible for managing Docker Containers of MESG services.
 // it can be implemented for any container orchestration tool.
+// TODO(ilgooz): discuss if these should accept service.Instance instead of service.Service.
```

Comment:
@mesg-foundation/core please provide feedbacks about this.

Reply:
Yes it should accept instance and I think this function can be moved directly to the sdk under the instance sdk

Reply:
I'll do this improvement in an another PR

Reply:
Yes it should accept instance instead of service.
Agree also to move the manager package directly inside sdk/instance.